releaseNotes/SpringCloudAzureFeatureManagement.md

```diff
@@ -6,6 +6,27 @@
 
 [Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Samples web][samples_web] | [Product documentation][docs]
 
+## 1.1.2/1.2.2 - February 24, 2020
+
+* `FeatureManager` now has method `getAllFeatureNames` which will return the names of all loaded Feature Flags.
+* Simplified Feature Management config file format. Example:
```

Comment:
It's not the file format change. It's schema.

This may be a good place to mention your new schema file.